src/main/java/uk/gov/companieshouse/model/db/dissolution/DissolutionData.java

```diff
@@ -1,19 +1,21 @@
-package uk.gov.companieshouse.model.db;
+package uk.gov.companieshouse.model.db.dissolution;
 
 import java.util.List;
 
 public class DissolutionData {
 
-    private String eTag;
+    private String etag;
```

Comment:
if you are changing any field names make sure that all confluence data model definitions are updated to reflect this.

Reply:
In fact, the field names were changed to match the confluence page. https://companieshouse.atlassian.net/wiki/spaces/ER/pages/1765638323/Dissolution+Data+Model+Definition